CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 * 22.0.0
-  - Deprecated: `x_init` removed from the initialisation of the Algorithms and `initial` will be used.
+  - Removal of deprecated code
+    - AcquisitionGeometry constructor removed, use factory `create` methods instead
```

Comment:
Of course you haven't removed the [constructor](https://docs.python.org/3/reference/datamodel.html#object.__new__), but the [`__init__` ](https://docs.python.org/3/reference/datamodel.html#object.__init__)method does not set the state of the instance to a working geometry now. To get a working geometry it's necessary to use the factory or do multiple set calls.